jax/_src/numpy/lax_numpy.py

```diff
@@ -1832,10 +1832,50 @@ def _moveaxis(a: Array, source: tuple[int, ...], destination: tuple[int, ...]) -
   return lax.transpose(a, perm)
 
 
-@util.implements(np.isclose)
 @partial(jit, static_argnames=('equal_nan',))
 def isclose(a: ArrayLike, b: ArrayLike, rtol: ArrayLike = 1e-05, atol: ArrayLike = 1e-08,
             equal_nan: bool = False) -> Array:
+  r"""Check if the elements of two arrays are approximately equal within a tolerance.
+
+  JAX implementation of :func:`numpy.allclose`.
+  
```

Comment:
Looks like there's some extra whitespace here which is making the linter sad!

Reply:
yes, whitespace fixed.